src/modules/launcher/Wox.Infrastructure/Wox.Infrastructure.csproj

```diff
@@ -12,7 +12,7 @@
     <Version>$(Version).0</Version>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
-    <OutputPath>..\..\..\..\$(Platform)\$(Configuration)\modules\launcher\WoxInfrastructure</OutputPath>
+    <OutputPath>..\..\..\..\$(Platform)\$(Configuration)\WoxInfrastructure</OutputPath>
```

Comment:
same here: Why separate dir ? I guess some conflicts happen, just curious to know why

Reply:
No special reason, just letting pt run keep the same internal structure as it had before when building.
WoxInfrastructure is a static lib and not used for release. If you check the installer result, that folder is not even there. I'm guessing there's no need to clean it now, as I'm not sure if some tests or build steps might depend on it.